drivers/lora/sx1276.c

```diff
@@ -458,67 +446,53 @@ const struct Radio_s Radio = {
 	.SetTxContinuousWave = SX1276SetTxContinuousWave,
 };
 
-static int sx1276_antenna_configure(void)
+static inline int __sx1276_configure_pin(struct device **dev,
+					 const char *controller,
+					 gpio_pin_t pin, gpio_flags_t flags)
 {
-	int ret = 0;
-
-#if DT_INST_NODE_HAS_PROP(0, antenna_enable_gpios)
-	dev_data.antenna_enable = device_get_binding(
-			DT_INST_GPIO_LABEL(0, antenna_enable_gpios));
-	if (!dev_data.antenna_enable) {
-		LOG_ERR("Cannot get pointer to %s device",
-		       DT_INST_GPIO_LABEL(0, antenna_enable_gpios));
+	*dev = device_get_binding(controller);
+	if (!(*dev)) {
+		LOG_ERR("Cannot get pointer to %s device", controller);
 		return -EIO;
 	}
 
-	ret = gpio_pin_configure(dev_data.antenna_enable,
-				 GPIO_ANTENNA_ENABLE_PIN,
-				 GPIO_OUTPUT_INACTIVE |
-					GPIO_ANTENNA_ENABLE_FLAGS);
-#endif
+	return gpio_pin_configure(*dev, pin, flags);
```

Comment:
Might be worth adding an error message here instead of a silent failure.

Reply:
Error message added.